src/west/manifest.py

```diff
@@ -1651,7 +1658,7 @@ def _load_project(self, pd: Dict, url_bases: Dict[str, str],
         path = (ctx.path_prefix / pfx / pd.get('path', name)).as_posix()
 
         ret = Project(name, url, pd.get('revision', defaults.revision),
-                      path, clone_depth=pd.get('clone-depth'),
+                      path, d.get('submodules'), clone_depth=pd.get('clone-depth'),
```

Comment:
TYPO: pd.get
Better to assign to proper attribute:
submodules=pd.get('submodules')

Reply:
How does it work then :smile:?

Reply:
Wow, I don't know :) I think that I did mistake when cleaning code, but maybe some results where cached and that's why this part wasn't executed and I didn't notice that it will fail.